framework/include/media/MediaPlayer.h

```diff
@@ -40,14 +40,14 @@ namespace media {
  * @details @b #include <media/MediaPlayer.h>
  * @since TizenRT v2.0 PRE
  */
-typedef enum player_result_e {
+typedef enum player_error_e : int {
 	/** MediaPlayer Error case */
 	PLAYER_ERROR_INVALID_OPERATION = -2,
 	PLAYER_ERROR_INVALID_PARAM = -1,
 	PLAYER_ERROR,
 	/** MediaPlayer Success case */
 	PLAYER_OK
-} player_result_t;
+} player_error_t;
```

Comment:
I'll raise another PR after #2038 is merged.
I'll follow with `recorder_error_t`

Reply:
Please, refer to wifi_manager_result_e in wifi_manager.h
For consistency, player_return_t might be better than player_error_t.
Also, instead of PLAYER_ERROR_INVALID_OPERATION, PLAYER_INVALID_OPERATION is much simpler and straightforward.